### PR TITLE
python3Packages.{cryptolyzer,cryptoparser}: init at {0.8.1,0.8.0}

### PR DESCRIPTION
**Pull request by Kranzes**
###### Description of changes
- [python3Packages.cryptoparser: init at 0.8.0](https://github.com/NixOS/nixpkgs/commit/4e94825dd2c12e4e47f210d6885f2d47d6a8fe21)
- [python3Packages.cryptolyzer: init at 0.8.1](https://github.com/NixOS/nixpkgs/commit/b8e47201f8b3d2a35858f9af72de7d8e515a1557)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Kranzes** [comment]
Result of `nixpkgs-review pr 185456` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.cryptolyzer</li>
    <li>python310Packages.cryptoparser</li>
    <li>python39Packages.cryptolyzer</li>
    <li>python39Packages.cryptoparser</li>
  </ul>
</details>

File: pkgs/development/python-modules/cryptolyzer/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  propagatedBuildInputs = [
+    certvalidator
+    attrs
+    six
+    urllib3
+    cryptoparser
+    requests
+  ];
+
+  doCheck = false; # Tests require networking
```
Comment (risicle) [inline pkgs/development/python-modules/cryptolyzer/default.nix:31]: Could still add a `pythonImportsCheck`.

File: pkgs/development/python-modules/cryptoparser/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    sha256 = "sha256-kJg8d1PoGIC0feefbJM8oyXcRyMGdg1wWkQUl/nSNCo=";
+  };
+
+  propagatedBuildInputs = [
+    attrs
+    six
+    asn1crypto
+    python-dateutil
+  ];
+
```
Comment (risicle) [inline pkgs/development/python-modules/cryptoparser/default.nix:25]: Again could use a `pythonImportsCheck`

File: pkgs/development/python-modules/cryptolyzer/default.nix (hunk 1 above)
Comment (Kranzes) [reply pkgs/development/python-modules/cryptolyzer/default.nix:31]: What do we gain from that?

Comment (risicle) [reply pkgs/development/python-modules/cryptolyzer/default.nix:31]: We avoid the case where we end up with a package that we can't even import successfully. It happens.

File: pkgs/development/python-modules/cryptoparser/default.nix (hunk 2 above)
Comment (Kranzes) [reply pkgs/development/python-modules/cryptoparser/default.nix:25]: done.

**Comment by Kranzes** [comment]
@NixOS/ofborg build

**Comment by Kranzes** [comment]
Result of `nixpkgs-review pr 185456` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python310Packages.cryptolyzer</li>
    <li>python310Packages.cryptoparser</li>
    <li>python39Packages.cryptolyzer</li>
    <li>python39Packages.cryptoparser</li>
  </ul>
</details>

**Review by wiredhikari — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
@ofborg build python3Packages.cryptolyzer python3Packages.cryptoparser

**Pull request merged**